### ISSUE TITLE
Nextcloud Zammad Integration Issue

### ISSUE THREAD

**Issue by Bahenaas** [nextcloud/integration_zammad#51]
Hello Hivemind,
trying to connect two docker images on the same Network.

For the Personal connection it tells me java script is not running.

`Failed to save Zammad options:

Nextcloud

<div id="nojavascript">
<div>
This application requires JavaScript for correct operation. Please <a href="https://www.enable-javascript.com/" target="_blank" rel="noreferrer noopener">enable JavaScript</a> and reload the page. </div>
</div>

Nextcloud

Internal Server Error
The server was unable to complete your request.
If this happens again, please send the technical details below to the server administrator.
More details can be found in the server log.

Technical details

Remote Address: {my browser ip}
Request ID: {something}

Nextcloud – a safe home for all your data `

Basically the same issue that was opened here on Aug. 29 by RTechSn 

Tried changing settings, reinstalling the images completely fresh but the issue persists.

For the oAuth method Zammad forces me to use a https adress wich is not nessecary since communication only is on the same network.

Found that several people have these issues but couldn't find any solution yet. 

Please help :)

**Comment by Bahenaas** [comment]
Using Nextcloud V 26.0.0 and Zammad 6.0.0

**Comment by marcelklehr** [comment]
> Basically the same issue that was opened here on Aug. 29 by RTechSn

Closing as a duplicate.
